### ISSUE TITLE
Duda con id_cliente y tabla compras grupo par

### ISSUE THREAD

**Issue by agustinapena** [IIC2413/Syllabus-2023-1#145]
Hola! Estamos teniendo ciertos problemas de integridad de datos al momento de poblar la tabla "compras". Esto es porque en compras.csv existe una columna que corresponde a id_cliente. Segun la base de datos entregada, en esta columna se encuentran valores como 76, 55, 92 etc. Sin embargo, al ver clientes.csv la columna id_cliente tiene valores desde el 100 hacia arriba. Esto genera una gran inconsistencia y sería un error difícil de arreglar sin generar pérdidas o "contaminación" de información. ¿Qué solución nos pueden dar?

**Comment by RICARDOOVIEDO** [comment]
Sin mayor explicación no entiendo que problema les genera tener valores de mas de 100. Si es que en la tabla clientes hay ciertos clientes que no hay en compras, simplemente no existe relación. 

Eso es lo que entiendo del problema, o es otra cosa?

**Comment by agustinapena** [comment]
No, no es eso a lo que me refiero. Lo que pasa es que en la tabla compras hay compras efectuadas por clientes con id's como el número 55 o el número 56. Sin embargo, en la tabla clientes no existen clientes con ese id por lo que es inconsistente. Además, aunque quisiéramos agregar filas a la tabla clientes para considerar a los clientes que faltan no podríamos hacerlo ya que no tenemos datos como la calle, el numero de domicilio ni la comuna/region. Espero que ahí quede más claro :)

**Comment by Clemente231199** [comment]
Tengo el mismo problema. No puedo completar la información de la tabla de compra debido a las relaciones entre las tablas.

**Comment by Paldana99** [comment]
Esto se debe a un error, por lo tanto, pueden corregirlo de la manera que mejor les convenga. Pueden dejar anotada la supuesta corrección en la entrega para que el corrector esté al tanto.
